OpenRA.Game/Platform.cs

```diff
@@ -102,14 +102,14 @@ public static string ResolvePath(string path)
 			path = path.TrimEnd(new char[] { ' ', '\t' });
 
 			// paths starting with ^ are relative to the support dir
-			if (path.StartsWith("^"))
+			if (path.StartsWith("^", StringComparison.InvariantCulture))
```

Comment:
I just read up on what `StringComparison.InvariantCulture` does, and I think an ordinal comparison would be more appropriate here. Why did you go with this instead?